agent/flow-trace/03_E3_REQUEST_AND_COMMITTEE.md

```
@@ -127,11 +129,16 @@ Requester calls: Interfold.request({
 
 ## Step 2: Sortition — Committee Selection (Rust-Side)
 
-When the running ciphernodes detect `E3Requested` and `CommitteeRequested` events from the chain:
+When the running ciphernodes detect `DkgFoldAttestationContextEstablished`, `E3Requested`, and
+`CommitteeRequested` events from the chain:
 
-### 2a. E3Requested Event Processing
+### 2a. Request Event Processing
 
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)